src/overlays/actors/ovl_Sample/ac_sample.c

```diff
@@ -1,54 +1,234 @@
 #include "ac_sample.h"
 #include "m_actor_dlftbls.h"
 #include "m_object.h"
+#include "m_rcp.h"
+#include "69E2C0.h"
+#include "m_msg_main.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
 
-void Ac_Sample_Actor_ct(Actor* thisx, Game_Play* game_play);
-void Ac_Sample_Actor_dt(Actor* thisx, Game_Play* game_play);
-void Ac_Sample_Actor_main(Actor* thisx, Game_Play* game_play);
-void Ac_Sample_Actor_draw(Actor* thisx, Game_Play* game_play);
+void Ac_Sample_Actor_ct(Actor* thisx, Game_Play* play);
+void Ac_Sample_Actor_dt(Actor* thisx, Game_Play* play);
+void Ac_Sample_Actor_main(Actor* thisx, Game_Play* play);
+void Ac_Sample_Actor_draw(Actor* thisx, Game_Play* play);
 
-#if 0
-ActorProfile Sample_Profile = {
-    /* */ ACTOR_SAMPLE,
-    /* */ ACTOR_PART_4,
-    /* */ 0,
-    /* */ 0x9000,
-    /* */ OBJECT_12,
-    /* */ sizeof(Sample),
-    /* */ Ac_Sample_Actor_ct,
-    /* */ Ac_Sample_Actor_dt,
-    /* */ Ac_Sample_Actor_main,
-    /* */ Ac_Sample_Actor_draw,
-    /* */ NULL,
-};
-#endif
+// TODO: extract assets
+
+extern BaseSkeletonR cKF_bs_r_hnw;
+extern BaseAnimationR cKF_ba_r_hnw_move;
+
+extern u8 hnw_tmem_txt[];
+extern u16 hnw_face[];
+
+void Ac_Sample_ct_forCorect(Actor* thisx, Game_Play* play) {
+    static ClObjPipe_Init Ac_Sample_OcInfoData_forStand = {
+        { OC1_1 | OC1_TYPE_8 | OC1_TYPE_10 | OC1_TYPE_20, OC2_TYPE_20, COLSHAPE_PIPE },
+        { ELEM_FLAG_1 },
+        { { 0x1E, 0x3C, 0, { 0, 0, 0 } } },
+    };
```

Comment:
This isn't static in the linker map, so for those they should be put outside of the function and non static.

Reply:
`52] Ac_Sample_OcInfoData_forStand (object,local) found in ac_sample.o`
Should be static outside the function, but unsure if it should be static at that point here

Reply:
I guess the benefit of keeping it non static is that it will show up in the map file. Doing in function static was mostly a concession so we don't have to rename a whole bunch of original names, but outside of the function you shouldn't do static. I apologize for this being confusing.